src/index.js

```diff
@@ -35,6 +36,10 @@ export default class SaucelabsConnector {
             retries:    WEB_DRIVER_CONFIGURATION_RETRIES,
             timeout:    WEB_DRIVER_CONFIGURATION_TIMEOUT
         });
+
+        this.options = {
+            showBrowserStartMessage: options.showBrowserStartMessage

```

Comment:
[Example](https://saucelabs.com/beta/tests/690ceb19d8a84ee98650e08e15b01e57/commands)
In other words we planning to add more extra options